Dockerfile

```diff
@@ -32,6 +32,12 @@ RUN set -x && \
     dnf -y upgrade; \
     dnf -y distro-sync --repo copr:copr.fedorainfracloud.org:rpmsoftwaremanagement:dnf5-unstable;
 
+RUN set -x && \
+    if [ -n "$COPR" ] && [ -n "$RPMS" ]; then \
```

Comment:
Nitpick - for clarity I'd prefer to rename RPMS -> COPR_RPMS. At first glance it looks like independent variables and using logical and was not clear to me.

Reply:
sure, seems logical to me